imports/plugins/core/layout/client/components/footer.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+import { registerComponent } from "/imports/plugins/core/components/lib";
+import { Reaction } from "/client/api";
+
+const Footer = () => (
+  <div className="reaction-navigation-footer footer-default">
+    <nav className="navbar-bottom" role="navigation">
+      <div className="row">
+        <a href={Reaction.Router.pathFor("/")}>Home</a>
```

Comment:
Remove this link for now. This template is valuable to have from a `replaceComponent` standpoint, but the link shows up and doesn't look good.

The below is the profile page.

<img width="872" alt="account_profile" src="https://user-images.githubusercontent.com/42217/32902783-300f9b14-caa8-11e7-98e5-15f3448c2f42.png">

Reply:
ok, removed the link from the component.